### PR TITLE
Integrate data

### PR DESCRIPTION
**Pull request by Wilfongjt**
Switch data source form San Francisco's open data portal to Citizenlabs open data portal.
Integrates Grand River Basin data into adopt-a-drain.

### CODE REVIEW

File: lib/thing_importer.rb [hunk 1]
```diff
@@ -1,52 +1,70 @@
...
+
+      return norm
     end
 
     def invalid_thing(thing)
-      false unless ['Storm Water Inlet Drain', 'Catch Basin Drain'].include?(thing[:type])
-      false unless integer?(thing[:city_id])
-      true
+      # puts 'invalid_thing 1'
+      rc = false
```
Comment (Ross-Hunter) [inline lib/thing_importer.rb]: It looks like this line isn't doing anything.

File: lib/thing_importer.rb [hunk 2]
```diff
@@ -1,52 +1,70 @@
...
+      false unless ['Storm Water Inlet Drain', 'Catch Basin Drain'].include?(thing["type"])
+      false unless integer?(thing["city_id"])
```
Comment (Ross-Hunter) [inline lib/thing_importer.rb]: These 2 lines aren't doing anything either. I think they are supposed to be `return false unless...`

Ruby has implicit return, but that only works on the last branch of any block. Below there is an explicit `return true`, which could be re-written as just `true` -- though it's fine to explicitly return too.

**Comment by Ross-Hunter** [comment]
I would not recommend commenting out the tests and instead suggest you fix them. There also is a lot of commented out code scattered about that will make this pretty hard to maintain long-term.

That being said, I'll defer to @jacebrowning about having the drop in test coverage fail the build. I traditionally have not found that to be a great use of everyone's time.

File: lib/thing_importer.rb (hunk 2 above)
Comment (Wilfongjt) [reply lib/thing_importer.rb]: Prefer explicit return.

File: test/test_helper.rb [hunk 3]
```diff
@@ -2,7 +2,7 @@
 
 require 'simplecov'
 require 'coveralls'
-require 'minitest/mock'
```
Comment (Ross-Hunter) [inline test/test_helper.rb]: This was needed for tests to work.

**Pull request merged**